### DISCUSSION TITLE
Autocompletion not triggered automatically with Volar

### DISCUSSION THREAD

**Discussion by pawelblaszczyk5** [unocss/unocss#1500] [Q&A]
Hello,
I have a strange problem with autocompletion and was wondering whether is a bug or common behaviour. When I'm writing in Vue files I get class suggestions only if I trigger it manually (CTRL + space). Firstly I checked how it works in a [playground](https://uno.antfu.me/play/) and noticed it auto suggests after few keystrokes. Then I noticed it works the same in my vscode in HTML files. I tried to dive into this and found out that it works the same way in Vue files if I disable Volar. 

Does anyone have the same problem and found some solution? Should I raise an issue or is it intended/upstream?

**Comment by ItsRyanWu** [comment]
Same issue here

**Comment by momoathome** [comment]
@ItsRyanWu see if the newest Version of UnoCss fixes your Problem with the autocompletion
https://github.com/unocss/unocss/pull/2769
